src/routes/github/create-branch/github-create-branch-router.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from "express";
+import { GithubCreateBranchGet } from "routes/github/create-branch/github-create-branch-get";
+
+export const GithubCreateBranchRouter = Router();
+
+GithubCreateBranchRouter.route("/")
+	.get(GithubCreateBranchGet);
+
```

Comment:
you don't need to add a router for a single route, just add it to the parent router.

Reply:
Did it in a separate route so that the POST request later can simply be appended here.

Reply:
ah right